src/status_im/wallet/accounts/core.cljs

```diff
@@ -87,6 +87,16 @@
             (navigation/navigate-to-cofx :account-added nil)))
 
 (fx/defn save-account
+  {:events [:wallet.accounts/save-account]}
+  [{:keys [db] :as cofx} account {:keys [name color]}]
+  (let [{:keys [accounts]} (:multiaccount db)
+        new-account (cond-> account
```

Comment:
Maybe we can do `(merge account new-account)` here to avoid `name`/`color` duplication.